src/adaptations/device-layer/include/Weave/DeviceLayer/ConfigurationManager.h

```diff
@@ -77,7 +78,12 @@ class ConfigurationManager
             uint8_t & hour, uint8_t & minute, uint8_t & second);
     WEAVE_ERROR GetDeviceId(uint64_t & deviceId);
     WEAVE_ERROR GetDeviceCertificate(uint8_t * buf, size_t bufSize, size_t & certLen);
+    WEAVE_ERROR GetDeviceICACerts(uint8_t * buf, size_t bufSize, size_t & certsLen);
```

Comment:
Generally I like to spell out names in the Device Layer API.  So I suggest changing this to GetDeviceIntermediateCACerts().